pkgs/tools/typesetting/sile/default.nix

```diff
@@ -109,6 +109,7 @@ stdenv.mkDerivation rec {
     '';
     homepage = "https://sile-typesetter.org/";
     platforms = platforms.unix;
+    broken = stdenv.isDarwin;   # https://github.com/NixOS/nixpkgs/issues/23018
```

Comment:
@marsam how is this related to [the build errors we got on darwin](https://github.com/NixOS/nixpkgs/pull/105438#issuecomment-735791712)?

Reply:
The error happens during the checkPhase because it tries to call the `sile` executable, which shebang points to the luaEnv wrapper, hence it refers to that issue.